provider/cluster/kube/client.go

```diff
@@ -115,19 +112,17 @@ func openKubeConfig(log log.Logger) (*rest.Config, error) {
 	return rest.InClusterConfig()
 }
 
-func (c *client) shouldExpose(expose *manifest.ServiceExpose) bool {
-	return expose.Global &&
-		(expose.ExternalPort == 80 ||
-			(expose.ExternalPort == 0 && expose.Port == 80))
+func shouldExpose(expose *manifest.ServiceExpose) bool {
+	return expose.Proto == manifest.TCP && expose.Global && 80 == exposeExternalPort(expose)
```

Comment:
make an issue or some note that only `TCP` ports can be exposed, some protocols like `QUIC` are based on `UDP` and won't work

Reply:
I don't know that kubernetes ingress supports QUIC at this time? HTTP 2.0 should work in theory.

Maybe QUIC support should be an issue down the road? I don't think it blocks any of the present use cases.